unity-client/Assets/Tutorial/Tests/TutorialControllerShould.cs

```diff
@@ -0,0 +1,181 @@
+using DCL.Tutorial;
+using NUnit.Framework;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.TestTools;
+
+namespace DCL.Tutorial_Tests
+{
+    public class TutorialControllerShould : TestsBase
```

Comment:
Judging for the pretty well encapsulated tests, maybe we can avoid the `TestsBase` inheritance and make a custom `SetUp` that just gets the tutorial controller ready?

Reply:
As you said, I have managed to avoid the use of TestsBase and now the tutorial tests are completely auto-managed and independent of any helper and any predefined scene :) Thanks for the advice!